### PR TITLE
apimon: Add dll_discovered event

### PR DESCRIPTION
**Pull request by BonusPlay**

### CODE REVIEW

File: src/plugins/apimon/apimon.cpp [hunk 1]
```diff
@@ -293,6 +293,19 @@ static void on_dll_discovered(drakvuf_t drakvuf, const std::string& dll_name, co
...
     apimon* plugin = (apimon*)extra;
 
+    vmi_pid_t pid;
+    {
+        auto vmi = vmi_lock_guard(drakvuf);
+        vmi_dtb_to_pid(vmi, dll->dtb, &pid);
+    }
+
+    fmt::print(plugin->m_output_format, "apimon", drakvuf, nullptr,
+        keyval("Event", fmt::Qstr("dll_discovered")),
```
[the remark this answers is not in the document]
Comment (BonusPlay) [reply src/plugins/apimon/apimon.cpp]: fixed

File: src/plugins/apimon/apimon.cpp [hunk 2]
```diff
@@ -293,6 +293,19 @@ static void on_dll_discovered(drakvuf_t drakvuf, const std::string& dll_name, co
...
+        keyval("DllName", fmt::Qstr(dll_name)),
```
[the remark this answers is not in the document]
Comment (BonusPlay) [reply src/plugins/apimon/apimon.cpp]: fixed

**Comment by BonusPlay** [comment]
I've also fixed one already existing log.

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Review by disaykin — APPROVED** [review]

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by BonusPlay** [comment]
Bump as this is a blocker for DRAKVUF-Sandbox. This PR literally adds one log point.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Pull request merged**